dll/win32/devmgr/lang/ro-RO.rc

```diff
@@ -170,7 +170,7 @@ END
 
 IDD_DEVICEDRIVER DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Module-pilot"
+CAPTION "drivere"
```

Comment:
```suggestion
CAPTION "Drivere"
```
all other captions do start with a capital letter. Please stick to that or it will look odd.